GoogleTestAdapter/Core/GoogleTestDiscoverer.cs

```diff
@@ -117,8 +117,9 @@ public bool IsGoogleTestExecutable(string executable, string customRegex = "")
                 regex = customRegex;
                 matches = SafeMatches(executable, customRegex);
             }
-            _logger.DebugInfo(
-                $"'{executable}' {(matches ? "matches" : "does not match")} {regexSource} regex '{regex}'");
+            string strMatch = " " + Resources.Matches + " " + regexSource + " " + String.Format(Resources.Regex, regex);
```

Comment:
That's weird that executable, regexSource, and regex are not just fill-ins in Resources.Matches/DontMatch.
Also it is wasteful to create two strings and use only one later on.